source/12_policies/mdqn.rst

```diff
@@ -0,0 +1,118 @@
+|                     |                 |                                                     |github.com/opendilab/     |  sdqn(3513)          |
+|                     |                 |                                                     |DI-engine/tree/main/dizoo/|                      |
+|Asterix              |  8963           |.. image:: images/benchmark/mdqn_asterix.png         |atari/config/serial/      |  paper(1718)         |
+|                     |                 |                                                     |asterix/asterix_mdqn_confi|                      |
+|(Asterix-v0)         |                 |                                                     |g.py>`_                   |  dqn(3444)           |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+|                     |                 |                                                     |`config_link_q <https://  |                      |
+|                     |                 |                                                     |github.com/opendilab/     |  sdqn(1804)          |
+|SpaceInvaders        |                 |                                                     |DI-engine/tree/main/dizoo/|                      |
+|                     |  2211           |.. image:: images/benchmark/mdqn_spaceinvaders.png   |atari/config/serial/      |  paper(2045)         |
+|(spaceInvaders-v0)   |                 |                                                     |spaceinvaders/spaceinvader|                      |
+|                     |                 |                                                     |s_dqn_config.py>`_        |  dqn(1228)           |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+|                     |                 |                                                     |`config_link_s <https://  |                      |
+|                     |                 |                                                     |github.com/opendilab/     |  sdqn(986.1)         |
+|Enduro               |                 |                                                     |DI-engine/tree/main/dizoo/|                      |
+|                     |  1003           |.. image:: images/benchmark/mdqn_enduro.png          |atari/config/serial/      |  paper(1171)         |
+|(Enduro-v4)          |                 |                                                     |endruo/space              |                      |
+|                     |                 |                                                     |enduro_mdqn_config.py>`_  |  dqn(986.4)          |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+
+
```

Comment:
add some naive explanation about the differences between and our config and paper config

Reply:
added